healthcareai/deploy_supervised_model.py

```diff
@@ -23,150 +25,152 @@ class DeploySupervisedModel(object):
     """
 
     def __init__(self,
-                 modeltype,
-                 df,
-                 graincol,
-                 windowcol,
-                 predictedcol,
+                 model_type,
+                 dataframe,
+                 grain_column,
+                 window_column,
+                 predicted_column,
                  impute,
                  debug=False):
-
-
-        self.modeltype = modeltype
-        self.df = df
-        self.graincol = graincol
+        self.modeltype = model_type
+        if model_type == 'classification':
+            self.predicted_column_name = 'PredictedProbNBR'
+        else:
+            self.predicted_column_name = 'PredictedValueNBR'
+        self.df = dataframe
+        self.grain_column = grain_column
         self.impute = impute
-        self.y_pred = None # Only used for unit test
+        self.y_pred = None  # Only used for unit test
 
         if debug:
             print('\nTypes of original dataframe:')
```

Comment:
One thing you might want to do in anticipation of setting-up logging is as follows.  Define this at the top of your module:
```
class PrintingLogger(object):
  def __init__(self, **kwargs):
    self.__dict__.update(kwargs)

LOG = PrintingLogger(debug=print, info=print, warning=print, error=print, critical=print)
```
Then you could replace all your 'print' statements with `LOG.info` or whatever severity is most apt for the message.

Then when we figure out how to share the logging code with hcai, we'll only have to change a small amount of code.  Just an idea!